pkg/cmd/server/api/v1/types.go

```diff
@@ -213,6 +213,9 @@ type NetworkConfig struct {
 	ClusterNetworkCIDR string `json:"clusterNetworkCIDR"`
 	HostSubnetLength   uint   `json:"hostSubnetLength"`
 	ServiceNetworkCIDR string `json:"serviceNetworkCIDR"`
+
+	// AdminNamespaces are the namespaces/projects that are allowed to reach all networks in case of multi-tenant SDN
+	AdminNamespaces []string `json:"adminNamespaces"`
```

Comment:
Is adminNamespaces the best term? Is "admin" the term used in sdn, or would "global" or "master" be better?